icons/mui.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>MUI</title><path d="M20.2286 15.7928a.66601.66601 0 0 0 .2444-.2432.6664.6664 0 0 0 .0902-.3328l.012-3.8581a.66622.66622 0 0 1 .0903-.3328.66601.66601 0 0 1 .2444-.2432l2.0914-1.2014a.66687.66687 0 0 1 .3331-.08855.66693.66693 0 0 1 .3329.08967.66703.66703 0 0 1 .2435.24392.66644.66644 0 0 1 .0892.33296v7.0142a.66659.66659 0 0 1-.3347.578l-7.8929 4.5335a.66629.66629 0 0 1-.6626.0006l-6.19422-3.5427a.6668.6668 0 0 1-.24586-.2441.6671.6671 0 0 1-.09015-.3346v-3.5368c0-.004.00467-.0066.00867-.0046.00333.002.008 0 .008-.0047v-.004c0-.0027.00133-.0053.004-.0067l5.10146-2.9307c.0047-.0027.0027-.01-.0027-.01a.00523.00523 0 0 1-.0037-.0016.00523.00523 0 0 1-.0016-.0037l.01-3.46748a.66656.66656 0 0 0-.3322-.57935.66657.66657 0 0 0-.6678-.00066L8.91225 9.79864a.66675.66675 0 0 1-.66535 0L4.44346 7.60857a.66668.66668 0 0 0-.99936.57735v6.26688a.66664.66664 0 0 1-.33182.5769.66612.66612 0 0 1-.33245.0902.6673.6673 0 0 1-.33309-.0878L.33601 13.8254a.66652.66652 0 0 1-.2462-.2447A.66682.66682 0 0 1 0 13.2454L.01866 2.29176a.6667.6667 0 0 1 .99869-.57668l7.2302 4.15278a.6667.6667 0 0 0 .66402 0l7.22822-4.15278a.66644.66644 0 0 1 .3333-.08855.6663.6663 0 0 1 .3329.08984.66712.66712 0 0 1 .2436.24419.66717.66717 0 0 1 .0889.3332V13.2501c0 .1172-.0308.2323-.0895.3338a.66671.66671 0 0 1-.2445.2442l-3.7855 2.18a.66709.66709 0 0 0-.2446.2448.66606.66606 0 0 0-.0891.3344.6671.6671 0 0 0 .0906.334.66608.66608 0 0 0 .2458.2435l2.0881 1.1887c.1009.0575.215.0876.3311.0874a.66703.66703 0 0 0 .3309-.0887l4.4568-2.5594Zm.438-9.82827a.66603.66603 0 0 0 .0909.3355.66628.66628 0 0 0 .2473.24416.66676.66676 0 0 0 .6711-.00831l2.0001-1.20004a.6662.6662 0 0 0 .2372-.243A.66658.66658 0 0 0 24 4.7645V2.31909a.6673.6673 0 0 0-.0911-.33526.66692.66692 0 0 0-.584-.33017.66692.66692 0 0 0-.3343.09475l-2 1.20004a.66646.66646 0 0 0-.2373.243.66755.66755 0 0 0-.0867.32834V5.9652v-.00067Z"/></svg>
```

Comment:
The icon is a perfect match! I'm just wondering if the precision of 5 was necessary?

Reply:
True, looks also good with a precision of 3 in my opinion :)